proto/viam/app/mltraining/ml_training.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+package viam.app.mltraining.v1;
+
+import "app/data/v1/data.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "go.viam.com/api/app/mltrainining/v1";
```

Comment:
nit: training spelled wrong here

Reply:
also: do we want to have this file in a folder called `v1` similar to datasync, data, etc..

Reply:
oops yes 100%, I accidentally made an empty v1 folder and left this outside of it, moved